src/sidebar/test/bootstrap.js

```diff
@@ -15,6 +15,10 @@ patch(assert);
 // karma-sinon plugins
 globalThis.assert = assert;
 globalThis.sinon = sinon;
+globalThis.specify ??= globalThis.it;
+globalThis.context ??= globalThis.describe;
+globalThis.before ??= globalThis.beforeAll;
+globalThis.after ??= globalThis.afterAll;
```

Comment:
A bunch of aliases for functions that are defined by mocha but not by vitest.

Except for `context`, which we use a lot and has its own meaning, the rest we could probably just replace them and remove the aliasing.